cmake can't find existing BLAS/LAPACK installation

<!--
Thanks for reporting an issue with deal.II! This HTML comment won't be visible
in the issue tracker but is present here to help you write a better description
of the problem you encountered.

If you have questions about deal.II, please post them on the mailing list:

   https://groups.google.com/g/dealii

Otherwise:
- Do you encounter a compilation failure? Please attach the detailed.log file
  present in the build directory here. Any additional compiler messages will also
  be helpful!

- Does something crash (e.g., an example) that shouldn't? Please include any relevant
  terminal output as well as the detailed.log file generated in your build
  directory (should you have one).

- Is there a missing feature you want to implement? Please briefly describe it and
  the problem it solves.

- Did you find some other issue with deal.II? Summarize the problem here.
-->

Trying to build dealii for my personal machine(Ubuntu 22.04 LTS) with Intel CORE(R) i7-9750H and NVIDIA GeForce GTX 1650. 

I am getting the following error when building with cmake. please see attached screenshot. 
![image](https://user-images.githubusercontent.com/57914003/208203163-8fe8b078-ee1d-44af-a47b-cd1df17591c3.png)
BLAS and LAPACK have been successfully built at /home/cruddybanana1102/Downloads/lapack-3.11.0/ 
but specifying that as file path while running cmake doesn't work. 

Details can be found in attached cmake output and error logs. 
[CMakeOutput.log](https://github.com/dealii/dealii/files/10249931/CMakeOutput.log)
[CMakeError.log](https://github.com/dealii/dealii/files/10249934/CMakeError.log)

May I ask how you called `cmake` precisely? I am asking because your CMakeError.log contains:
```
gmake[1]: *** No rule to make target '/home/cruddyanana1102/Downloads/lapack-3.11.0', needed by 'cmTC_31925'.  Stop.
```
In this regard @CodyGirl 's recommendation is spot on: It is best to compile shared libraries and hint via `LAPACK_DIR` (or `CMAKE_PREFIX_PATH`).



A quick comment: If your goal is to simply install deal.II (and you're fine with using software packaged in Ubuntu) then it's not necessary to compile and install any dependency by hand. You can either
 - install the current version of  deal.II from a PPA [1] by running `apt install libdeal.ii-dev`
 - or install all build dependencies (with or without above PPA - but with PPA is recommended from my side): `apt build-dep libdeal.ii-dev`

[1] https://launchpad.net/~ginggs/+archive/ubuntu/deal.ii-9.4.0-backports

--- a/doc/external-libs/lapack.html
+++ b/doc/external-libs/lapack.html
@@ -1,0 +1,108 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+	  "http://www.w3.org/TR/html4/loose.dtd">
+
+<html>
+  <head>
+    <title>The deal.II Readme on interfacing to BLAS/LAPACK</title>
+    <link href="../screen.css" rel="StyleSheet">
+    <meta name="copyright" content="Copyright (C) 2010 - 2017 by the deal.II authors">
+    <meta name="keywords" content="deal.II">
+  </head>
+  <body>
+
+    <h2>Installation of <acronym>BLAS/LAPACK</acronym></h2>
+
+    <p>
+      <a href="https://netlib.org/lapack/">LAPACK</a>
+      is a library of Fortran subroutines for solving the most
+      commonly occurring problems in numerical linear algebra.
+      LAPACK reference implementation is available on
+      <a href="https://github.com/Reference-LAPACK/lapack">GitHub</a>,
+      corresponding releases are available on
+      <a href="https://netlib.org/lapack/">netlib</a>.
+      It is worth noting that there are further implementations of BLAS/LAPACK
+      libraries, such as <a href="https://github.com/xianyi/OpenBLAS">OpenBLAS</a> and
+      <a href="https://www.intel.com/content/www/us/en/developer/tools/oneapi/onemkl.html#gs.m4jlqg">Intel MKL</a>,
+    </p>
+
+    <p>
+      <a href="https://netlib.org/lapack/">LAPACK</a> and
+      <a href="https://netlib.org/blas/">BLAS</a> are packaged and
+      distributed by distros such as Debian, Ubuntu, Pop!OS, CentOS and others.
+
+      It is recommended to install these packages using apt or yum package
+      managers. They can be installed on Debian or Ubuntu systems using
+      the following commands.
+      <pre>
+        sudo apt update && sudo apt upgrade
+        sudo apt install libblas3 libblas-dev
+        sudo apt install liblapack3 liblapack-dev
+      </pre>
+    </p>
+
+    <p>
+      Below is a short summary of instructions on how to compile and install
+      <acronym>LAPACK</acronym> by hand (for the case you wish to do so).
+    </p>
+
+    <h3>How to compile and install <acronym>LAPACK</acronym> by hand</h3>
+
+    <p>
+      First clone the <a href="https://github.com/Reference-LAPACK/lapack">LAPACK</a>
+      repository. And configure cmake to build lapack as a shared library.
+      The following commands will result in a set of .so files or shared library
+      files installed in the path specified with CMAKE_INSTALL_LIBDIR flag :
+
+      <pre>
+        cd lapack
+        mkdir build
+        cd build
+        cmake -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_LIBDIR=$HOME/.local/lapack ../
+        make install
+      </pre>
+
+      After successful execution of above set of commands, a list of .so files
+      such as libblas.so, libblas.so.3, liblapack.so, liblapack.so.3 are
+      generated.
+    </p>
+
+    <h2>Interfacing <acronym>deal.II</acronym>
+      to <acronym>LAPACK</acronym></h2>
+
+    <p>
+      Support for <acronym>LAPACK</acronym> will be
+      enabled automatically if a system wide installation of
+      <acronym>LAPACK</acronym> can be found.
+      To use a self compiled version, specify
+      <pre>
+    -DLAPACK_DIR=/path/to/lapack-installation
+      </pre>
+      when invoking <code>cmake</code>, for example,
+
+      <pre>
+        git clone https://github.com/dealii/dealii.git
+        cd dealii
+        mkdir build
+        cd build
+        cmake -DDEAL_II_WITH_LAPACK=ON -DLAPACK_DIR=$HOME/.local/lapack/ ../
+        make
+      </pre>
+
+    </p>
+    <p>
+      You can override the autodetection by manually setting
+      <pre>
+    -DDEAL_II_WITH_LAPACK=OFF|ON
+      </pre>
+    </p>
+
+    <hr />
+    <div class="right">
+      <a href="http://validator.w3.org/check?uri=referer" target="_top">
+        <img style="border:0" src="http://www.w3.org/Icons/valid-html401" alt="Valid HTML 4.01!"></a>
+      <a href="http://jigsaw.w3.org/css-validator/check/referer" target="_top">
+        <img style="border:0;width:88px;height:31px" src="http://jigsaw.w3.org/css-validator/images/vcss" alt="Valid CSS!"></a>
+    </div>
+
+  </body>
+</html>

--- a/doc/readme.html
+++ b/doc/readme.html
@@ -449,9 +449,8 @@
                 <a href="http://www.netlib.org/blas/" target="_top">BLAS</a> (the <i>Basic Linear Algebra Subroutines</i>) and
                 <a href="http://www.netlib.org/lapack/" target="_top">LAPACK</a> (
                 <i>Linear Algebra Package</i>) are two packages that support low-level linear algebra operations on vectors and dense matrices. Both libraries should be packaged by almost all Linux distributions and found automatically whenever available.
-                (You might have to install development versions of the libraries for <acronym>deal.II</acronym> being able to use them). For details on how to set up <acronym>deal.II</acronym> with a non standard BLAS/LAPACK implementation, see the
-                <a href="users/cmake_dealii.html#advanced">advanced
-            setup</a> section in the CMake ReadME.
+                (You might have to install development versions of the libraries for <acronym>deal.II</acronym> being able to use them). For details on how to set up <acronym>deal.II</acronym> with a non standard BLAS/LAPACK implementation,
+                see <a href="external-libs/lapack.html" target="body">this page</a>.
             </p>
         </dd>
 

